ush/python/pygfs/task/land_analysis.py

```diff
@@ -180,3 +320,168 @@ def get_bkg_dict(self, config: Dict) -> Dict[str, List[str]]:
             'copy': bkglist
         }
         return bkg_dict
+
+    @staticmethod
+    @logit(logger)
+    def get_ens_bkg_dict(config: Dict) -> Dict:
+        """Compile a dictionary of model background files to copy for the ensemble
+        Note that a "Fake" 2-member ensemble backgroud is being created by copying FV3 RESTART files (coupler, sfc_data)
+        from the deterministic background to DATA/bkg/mem001, 002.
+
+         Parameters
+         ----------
+         config: Dict
+             Dictionary of key-value pairs needed in this method
+             Should contain the following keys:
+             COM_ATMOS_RESTART_PREV
+             DATA
+             current_cycle
+             ntiles
+
+         Returns
+         ----------
+         bkg_dict: Dict
+             a dictionary containing the list of model background files to copy for FileHandler
+         """
+
+        dirlist = []
+        bkglist = []
+
+        # get FV3 sfc_data RESTART files; Note an ensemble is being created
+        rst_dir = os.path.join(config.COM_ATMOS_RESTART_PREV)
+
+        for imem in range(1, LandAnalysis.NMEM_LANDENS + 1):
+            memchar = f"mem{imem:03d}"
+
+            run_dir = os.path.join(config.DATA, 'bkg', memchar, 'RESTART')
+            dirlist.append(run_dir)
+
+            # Land DA needs coupler
+            basename = f'{to_fv3time(config.current_cycle)}.coupler.res'
+            bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+
+            # Land DA only needs sfc_data
+            for ftype in ['sfc_data']:
+                template = f'{to_fv3time(config.current_cycle)}.{ftype}.tile{{tilenum}}.nc'
+                for itile in range(1, config.ntiles + 1):
+                    basename = template.format(tilenum=itile)
+                    bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+
+        bkg_dict = {
+            'mkdir': dirlist,
+            'copy': bkglist
+        }
+
+        return bkg_dict
+
+    @staticmethod
+    @logit(logger)
+    def create_ensemble(vname: str, bestddev: float, config: Dict) -> None:
+        """Create an ensemble for Snow Depth analysis by perturbing snow depth with a prescribed variance.
+        Additionally, remove glacier locations
+
+        Parameters
+        ----------
+        vname : str
```

Comment:
For now this works, because we are only using the method for one variable. But if additional variables were ever added, loading the same NetCDF over and over is inefficient. On the other hand, if the the doc-block is correct and this should only be used for snow depth, perhaps the method name should reflect that: `create_snow_depth_ens()`, and then I'd be less worried about extending this to worry about multiple variables.

Reply:
Since the method is part of `land_analysis.py`, it is not necessary to say `create_snow_depth_ens`.
I would wait till there is more information on what other state variables will they perturb and how to create separate methods or turn this into an interface.